gulpfile.js

```diff
@@ -29,7 +29,7 @@
   });
 
   gulp.task('pre-test', function () {
-    return gulp.src('generators/**/*.js')
+    return gulp.src('generators/app/*.js')
```

Comment:
Both means the same for current file structure, your change makes it even more strict which is fine